R/setup_abm.R

```diff
@@ -26,8 +26,12 @@ setup_abm <- function(
   asymptomatic_relative_infectiousness = 0.5,
   vaccination_test_seeking_multiplier = 1,
   
-  # whether to do screening of symptomatics
-  screening = TRUE,
+  # whether to do routine screening at workplaces
+  workplace_screening = TRUE,
+  
+  # switch for passive presentation of symptomatic individuals
+  passive_detections = TRUE,
```

Comment:
I'd call this one `symptomatic_detections`

Reply:
replaced throughout